equinix/resource_metal_spot_market_request_acc_test.go

```diff
@@ -93,7 +93,7 @@ data "equinix_metal_spot_market_request" "dreq" {
 
 resource "equinix_metal_spot_market_request" "request" {
   project_id       = equinix_metal_project.test.id
-  max_bid_price    = data.equinix_metal_spot_market_price.test.price * 1.2
+  max_bid_price    = format("%%.2f", data.equinix_metal_spot_market_price.test.price)
```

Comment:
With this latest approach to passing in the original spot market price, both tests pass:
- https://github.com/equinix/terraform-provider-equinix/actions/runs/5081637323/jobs/9130308071?pr=334#step:5:1141
- https://github.com/equinix/terraform-provider-equinix/actions/runs/5081637323/jobs/9130308071?pr=334#step:5:1757